### PR TITLE
Add install instructions for the workspace catalog repo

### PR DESCRIPTION
**Pull request by cbuto**
## Description of changes being made

Adding docs about how to install the workspace catalog since this will be a manual task until https://jira.d2iq.com/browse/D2IQ-79950 is resolved. I just don't want this to get lost!

## Checklist

- [ ] Check with @takirala before release if this is needed

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md [hunk 1]
```diff
@@ -8,6 +8,40 @@ excerpt: Workspace Catalog Applications
...
+    spec:
+      interval: 1m0s
+      ref:
+        branch: main
+      timeout: 20s
+      url: https://github.com/mesosphere/dkp-catalog-applications  
+    EOF
+    ```
+
+1.  You should now see the workspace catalog `Apps` available in the UI and CLI
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md:39]:
```suggestion
1.  You should now see the workspace catalog `Apps` available in the UI, and in the CLI using `kubectl`:
```

**Review by msdolbey — APPROVED** [review]
I don't know if what I added is necessary, but it looks good either way
👍 
Lovely

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md (hunk 1 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md]: good call, fixed in ac0b90a0140ab050ffc007c599d0216ff412e534!

Comment (msdolbey) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md]: 👍

**Comment by cmastr** [comment]
Let's just make sure we take this back out when the other referenced item is merged then? Add something to the JIRA maybe. @cbuto

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md [hunk 2]
```diff
@@ -8,6 +8,40 @@ excerpt: Workspace Catalog Applications
 <!-- TODO: settle down the link target to custom applications -->
 Catalog applications are any third-party or open source applications that appear in the Catalog. These applications can be provided by D2iQ for use in your environment.
 
+## Installing the workspace catalog via the CLI
```
Comment (cmastr) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md:11]:
```suggestion
## Install the workspace catalog via the CLI
```

**Review by cmastr — APPROVED** [review]

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md (hunk 2 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md]: fixed in 10fe738!

**Comment by cbuto** [comment]
> Let's just make sure we take this back out when the other referenced item is merged then? Add something to the JIRA maybe. @cbuto

@cmastr definitely! I'll make sure this ends up in Jira somehow

**Review by cwyl02 — APPROVED** [review]

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md [hunk 3]
```diff
@@ -8,6 +8,40 @@ excerpt: Workspace Catalog Applications
 <!-- TODO: settle down the link target to custom applications -->
 Catalog applications are any third-party or open source applications that appear in the Catalog. These applications can be provided by D2iQ for use in your environment.
 
+## Install the workspace catalog via the CLI
+
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md:12]:
```suggestion

Follow these steps to install the catalog from the CLI.

```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md [hunk 4]
```diff
@@ -8,6 +8,40 @@ excerpt: Workspace Catalog Applications
...
+    spec:
+      interval: 1m0s
+      ref:
+        branch: main
+      timeout: 20s
+      url: https://github.com/mesosphere/dkp-catalog-applications  
+    EOF
+    ```
+
+1.  You should now see the workspace catalog `Apps` available in the UI, and in the CLI using `kubectl`:
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md:39]:
```suggestion
1.  Verify that you can see the workspace catalog `Apps` available in the UI and in the CLI, using `kubectl`:
```

**Review by Colin1841 — APPROVED** [review]

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md (hunk 4 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md]: fixed in 56cbc97!

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md (hunk 3 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md:12]: fixed in 56cbc97

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md [hunk 5]
```diff
@@ -8,6 +8,42 @@ excerpt: Workspace Catalog Applications
...
+    kind: GitRepository
+    metadata:
+      name: dkp-catalog-applications
+      namespace: ${WORKSPACE_NAMESPACE}
+      labels:
+        kommander.d2iq.io/gitapps-gitrepository-type: catalog
+    spec:
+      interval: 1m0s
+      ref:
+        branch: main
```
Comment (gracedo) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md:35]:
```suggestion
        tag: v2.1.1
```
I think we will be tagging dkp-catalog-applications so lets pin to the release version

Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md]: gotcha, fixed in 5d2bcc7

**Comment by cbuto** [comment]
Add some AC to this ticket to revisit this documentation: https://jira.d2iq.com/browse/D2IQ-79950

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md [hunk 6]
```diff
@@ -8,6 +8,43 @@ excerpt: Workspace Catalog Applications
...
+    ```sh
+    kubectl apply -f - <<EOF
+    apiVersion: source.toolkit.fluxcd.io/v1beta1
+    kind: GitRepository
+    metadata:
+      name: dkp-catalog-applications
+      namespace: ${WORKSPACE_NAMESPACE}
+      labels:
+        kommander.d2iq.io/gitapps-gitrepository-type: catalog
+        kommander.d2iq.io/gitrepository-type: catalog
```
Comment (cbuto) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md:32]: @takirala I added the recently required label, I think we still need both right?

Comment (gracedo) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md:32]: Yep! We need both for our dkp repo. For user-created custom apps, we just need the gitrepository-type label (gotta add that one to the docs for custom apps)

Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md:32]: perfect!

**Pull request merged**